src/core/client/ui/hooks/useDateTimeFormat.ts

```diff
@@ -1,11 +1,9 @@
 import { useMemo } from "react";
 
+import { createDateFormatter } from "coral-common/date";
 import { useUIContext } from "coral-ui/components/v2";
 
 export default function useDateTimeFormat(options: Intl.DateTimeFormatOptions) {
   const { locales } = useUIContext();
-  return useMemo(() => new Intl.DateTimeFormat(locales, options), [
-    locales,
-    options,
-  ]);
+  return useMemo(() => createDateFormatter(locales, options), [locales]);
```

Comment:
```suggestion
  return useMemo(() => createDateFormatter(locales, options), [locales, options]);
```

`options` must be included as a dependency in `useMemo` in case the value ever changes.